packages/zowe-explorer/src/trees/dataset/DatasetFSProvider.ts

```diff
@@ -625,10 +633,10 @@ export class DatasetFSProvider extends BaseProvider implements vscode.FileSystem
             throw vscode.FileSystemError.FileExists(uri);
         }
 
-        // Attempt to write data to remote system, and handle any conflicts from e-tag mismatch
```

Comment:
I don't think we should remove this comment because it provides context for the logic below